values/values.go

```diff
@@ -294,6 +294,11 @@ func CheckKind(got, exp semantic.Nature) {
 	}
 }
 
+// check if value is Timeable
+func IsTimeable(v Value) bool {
```

Comment:
I think this should be part of a different PR, but for general knowledge, it is a Go standard to start all documentation comments with `// NameOfFunction` as it forces the documenter to use the active voice and it makes it easier to grep for documentation comments with `git grep "// NameOfFunction"`.